packages/core-utils/src/fees.ts

```diff
@@ -1,6 +1,7 @@
 /**
  * Fee related serialization and deserialization
  */
+/* eslint prefer-arrow/prefer-arrow-functions: "off" */
```

Comment:
Instead of this could we either:

1. actually comply with the rule?
2. get away with a small number of rule-specific disable-next-line comments?